packages/fether-electron/src/main/index.js

```diff
@@ -4,7 +4,7 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
 import electron from 'electron';
-import parseUrl from 'parse-url';
```

Comment:
library can be removed from the package.json/yarn.lock as well

Reply:
fixed thanks